fuzzing/agentx_parse_fuzzer.c

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
If @fenner agrees, please drop the Google copyright statement and change this into a three-clause BSD license. I think this change is compatible with the oss-fuzz CLA since that CLA grants Google a license but it does not make Google the exclusive owner. See also https://cla.developers.google.com/about/google-individual.

Reply:
No objection to changing the copyright (and specifically grant copyright change for the code I originally authored).  The copyright in the oss-fuzz repo is requested to be the Google one - https://google.github.io/oss-fuzz/getting-started/new-project-guide/#copyright-headers - but that only covers the oss-fuzz repo.